src/main/java/games/strategy/engine/framework/GameDataUtils.java

```diff
@@ -49,7 +50,7 @@ public static Object translateIntoOtherGameData(final Object object, final GameD
       final GameObjectStreamFactory factory = new GameObjectStreamFactory(translateInto);
       final ObjectInputStream in = factory.create(source);
       try {
-        return in.readObject();
+        return (T) in.readObject();
```

Comment:
Just noting this for my own benefit...  This appears to be a safe cast because we know we're deserializing an object of the exact same type we just serialized a few lines above.